frontend/app/src/layouts/PreviewLayout/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, ReactNode } from 'react';
+import { PreviewLayoutContainer } from '@meethead/ui';
+import { ContactList } from '@meethead/ui';
+import { getContacts } from 'services/contacts';
+import { useQuery } from 'react-query';
+import { useTranslation } from 'react-i18next';
+
+type PreviewLayout = {
+  children?: ReactNode;
+};
+
+const PreviewLayout: FC<PreviewLayout> = () => {
+  const { t } = useTranslation(['config', 'translations']);
+  const { data: contacts } = useQuery(['getContacts'], async () => await getContacts('63120fd98028831ecf43b444'));
```

Comment:
You don't have defined Contact interface. The code should look like `const `{data: Contact[]}`.

Isn't this throwing a typescript error?

Reply:
I doesnt throw any errors and it works well like before, all i do was moving that from landing to that component